Gemfile

```diff
@@ -10,3 +12,6 @@ gemspec
 
 # Load Gemfile with dependencies from manageiq
 eval_gemfile(File.expand_path("spec/manageiq/Gemfile", __dir__))
+
+# load openapi gem
+gem 'openapi_client', :path => 'lib/autosde_oas_client/generated'
```

Comment:
openapi_client is too general. Many application use openapi.
 Please name this soemthing like autosde_openapi_client.

Reply:
While this is in the same repository I think we should leave it as a "normal" library rather than adding a path based gem.
We shouldn't be too far away from extracting this to a proper gem so I'd rather just wait for that.

@gregoryb-at-ibm is right though we should update the `.openapi-config.json` to take a `gemName` so it is something other than the default.

Reply:
I've generated openclient anew with name  'autosde_openapi_client'.